aiven/client/pretty.py

```diff
@@ -161,8 +154,9 @@ def iter_values(key, value):
         # The main, horizontal, line
         yield "  ".join(formatted_row.get(f, "").ljust(widths[f]) for f in horizontal_fields).strip()
         # And the rest of the fields, one per field
-        fields_to_print = []
-        for vertical_field in table_layout[1:]:
+        fields_to_print: List[Tuple[str, str]] = []
+        vertical_fields = cast(Iterator[str], itertools.islice(table_layout, 1, None))
```

Comment:
Is the `cast` really needed here?

Reply:
Without it, mypy can't narrow the iterable to just `Iterable[str]`, especially given how `table_layout` is annotated. When used in the for loop without casting, `vertical_field` would just be inferred as `List[str] | Tuple[str] | str`, yet it is assumed that all values aside from the first one in `table_layout`, are always `str` representing the vertical fields, if any.

Reply:
I was wondering if `vertical_fields: Iterator[str]` would work

Reply:
causes a conflict with `islice` generic types, seems to expect the first `__init__` argument to be of the same base type:
 ```
 aiven/client/pretty.py:158: error: Argument 1 to "islice" has incompatible type "Collection[Union[List[str], Tuple[str], str]]"; expected "Iterable[str]"
 ```